ui/raidboss/timeline.js

```diff
@@ -794,22 +794,67 @@ class TimelineController {
     this.dataFiles = {};
     // data files not sent yet.
     this.timelines = null;
+
+    // Fix for #1453
+    // Countdown state
+    // 0 = No countdown
+    // 1 = Countdown active
+    // 2 = Wiped during countdown
+    this.countdownState = 0;
+    this.wipeRegex = / (?:21:........:400000(?:10|03)|00:0038:cactbot wipe)/;
```

Comment:
`cactbot wipe` and `4000003` victory both don't reset the timeline (right now, at least).  It also seems unlikely that somebody would beat the fight while the countdown was active as you can't start the countdown mid-combat.  The only thing that stops the timeline is leaving combat or anything that jumps to zero.  (I'm open to these things changing, just telling it like it is.)

Can I suggest `this.wipeRegex = Regexes.network6d({ command: '40000010' });` instead?

Reply:
Actually, that's a good point. Does `cactbot wipe` even exist anymore?

I agree that just 40000010 makes sense.

Reply:
Changed for latest commit.